integration_tests/util.go

```diff
@@ -55,7 +55,7 @@ func LndHubTestServiceInit(lndClientMock lnd.LightningClientWrapper) (svc *servi
 		MaxReceiveAmount:      1000000,
 		MaxSendAmount:         100000,
 		LnurlAPIPrefix:        "testnet",
-		LnurlDomain:           "mintter.com",
+		LnurlDomain:           "domain.com",
```

Comment:
`example.com` is better since it is reserved - see https://en.wikipedia.org/wiki/Example.com - while http://domain.com is a commercial service

Reply:
done!